bin/pycbc_dark_vs_bright_injections

```diff
@@ -105,7 +116,7 @@ for i, inj in enumerate(injections.table):
         out_sim_inspiral_bright.append(inj)
     # NSBH systems
     elif numpy.logical_not(numpy.logical_and(m1 >= opts.ns_bh_boundary, m2 >= opts.ns_bh_boundary)):
-        eta = inj.eta
+        eta = inj.eta if inj.eta is not None else m1*m2/(m1+m2)**2
```

Comment:
Please use `pycbc.conversions.eta_from_mass1_mass2()` instead of the explicit equation.

Reply:
Done.